### PR TITLE
fix: removed stage from ws url

### PR DESCRIPTION
**Pull request by andreabadesso**
### Summary

We were appending the stage when assembling the URL for WebSocket lambda calls, this worked for the ApiGateway endpoints, as they have the stage on the URL but failed for custom domains without the stage path

### Acceptance Criteria
- Websocket connection should be able to call the lambdas that handle the responses without `Internal Server Errors`

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/ws/utils.ts [hunk 1]
```diff
@@ -18,11 +18,12 @@ export const connectionInfoFromEvent = (
...
     };
   }
+
   const domain = event.requestContext.domainName;
-  const stage = event.requestContext.stage;
+
   return {
     id: connID,
-    url: util.format(util.format('https://%s/%s', domain, stage)),
+    url: util.format('https://%s', domain),
```
Comment (luislhl) [inline src/ws/utils.ts:32]: Would it be better to receive this domain from an env var?

**Review by luislhl — APPROVED** [review]

**Review by r4mmer — APPROVED** [review]

File: src/ws/utils.ts (hunk 1 above)
Comment (andreabadesso) [reply src/ws/utils.ts:32]: It would, done in 75eb5ed18b40ddc506d69a444bf980750d1c6a04

File: .codebuild/buildspec.yml [hunk 2]
```diff
@@ -19,6 +19,9 @@ env:
     testnet_NETWORK: "testnet"
     mainnet_DEFAULT_SERVER: "https://wallet-service.private-nodes.hathor.network/v1a/"
     mainnet_NETWORK: "mainnet"
+    mainnet_WS_DOMAIN: "ws.wallet-service.hathor.network"
+    testnet_WS_DOMAIN: "ws.testnet.wallet-service.hathor.network"
+    dev_WS_DOMAIN: "ws.dev.wallet-service.testnet.hathor.network"
```
Comment (luislhl) [inline .codebuild/buildspec.yml]: I think we should stick with either grouping by env or grouping by variable. We are mixing both.

```
    dev_DEFAULT_SERVER: "https://dev.wallet-service.private-nodes.hathor.network/v1a/"
    dev_NETWORK: "testnet"
    dev_WS_DOMAIN: "ws.dev.wallet-service.testnet.hathor.network"
    testnet_DEFAULT_SERVER: "https://testnet.wallet-service.private-nodes.hathor.network/v1a/"
    testnet_NETWORK: "testnet"
    testnet_WS_DOMAIN: "ws.testnet.wallet-service.hathor.network"
    mainnet_DEFAULT_SERVER: "https://wallet-service.private-nodes.hathor.network/v1a/"
    mainnet_NETWORK: "mainnet"
    mainnet_WS_DOMAIN: "ws.wallet-service.hathor.network"
```

OR

```
    mainnet_DEFAULT_SERVER: "https://wallet-service.private-nodes.hathor.network/v1a/"
    testnet_DEFAULT_SERVER: "https://testnet.wallet-service.private-nodes.hathor.network/v1a/"
    dev_DEFAULT_SERVER: "https://dev.wallet-service.private-nodes.hathor.network/v1a/"
    mainnet_NETWORK: "mainnet"
    testnet_NETWORK: "testnet"
    dev_NETWORK: "testnet"
    mainnet_WS_DOMAIN: "ws.wallet-service.hathor.network"
    testnet_WS_DOMAIN: "ws.testnet.wallet-service.hathor.network"
    dev_WS_DOMAIN: "ws.dev.wallet-service.testnet.hathor.network"
```

Comment (andreabadesso) [reply .codebuild/buildspec.yml]: Wow, you are right, done in 87f0232d57816d2d7ebcddaaf8a675ddc8bacd7b

**Review by r4mmer — APPROVED** [review]

**Review by luislhl — APPROVED** [review]

**Pull request merged**